src/components/pages/client/BookingConfirmationPage.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="booking-confirmation-page">
+    <template v-if="seatsAndTickets.length >= 1">
+      <h3 class="booking-confirmation-page__title">Hell yeah!</h3>
+      <h4 class="booking-confirmation-page__subtitle">You booked 3 tickets</h4>
```

Comment:
If it's the number of tickets then I guess we could do
```suggestion
      <h4 class="booking-confirmation-page__subtitle">You booked {{ seatsAndTickets.length }} tickets</h4>
```

Reply:
of course, you are right! I forgot about it :facepalm: